atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -43,6 +43,11 @@ public int poolSize() {
         return 20;
     }
 
+    @Value.Default
+    public int poolRefreshIntervalSeconds() {
+        return 5 * 60;
```

Comment:
Do we track the values of timeouts/etc in CassandraConstants? We can set it back to 10 if people are potentially relying on this behavior. Mainly just wanted it configurable.